src/components/BookItem.js

```diff
@@ -1,43 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/Books/books';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
```

Comment:
- [ ] Kindly check the how you import this file and module, am getting the error below when running your app locally:
![image](https://user-images.githubusercontent.com/61408860/174095894-35f17a78-182b-4620-88af-b3428f70dbd6.png)